src/main/java/com/solvd/micro9/vkontaktah/web/LocalDateTimeToStringCoercing.java

```diff
@@ -0,0 +1,58 @@
+package com.solvd.micro9.vkontaktah.web;
+
+import graphql.language.StringValue;
+import graphql.schema.Coercing;
+import graphql.schema.CoercingParseLiteralException;
+import graphql.schema.CoercingParseValueException;
+import graphql.schema.CoercingSerializeException;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+@Component
+public class LocalDateTimeToStringCoercing implements Coercing<LocalDateTime, String> {
```

Comment:
@eyakauleva quite big class name

Reply:
@h1alexbel I know but I don't see any smaller appropriate name, cause smth like 'CustomCoercing' isn't that good because of the fact there can be a plenty of different custom coercing implementations

Reply:
@eyakauleva 
```java
class TimeAsString implements Coercing<>
```

Reply:
@h1alexbel this is a good one, I haven't thought about it, thanks!

Reply:
done 8897062